arxiv/base/routes.py

```diff
@@ -1,14 +1,61 @@
 """Provides routes for verifying base templates."""
```

Comment:
Would add to docstring note about verifying exceptions too.

Reply:
Will do